pyccel/stdlib/parallel/openacc.py

```diff
@@ -1,3 +1,8 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+# pylint: disable=missing-module-docstring
 # coding: utf-8
 
```

Comment:
move line 6 to the top of the file.
`coding utf` needs to be at the top of the file.